spec/gateways/route53_spec.rb

```diff
@@ -0,0 +1,9 @@
+describe Gateways::Route53 do
```

Comment:
Are you happy with just these specs? 

It looks like we're just testing the empty case, but not when there's actually stuff for it to retrieve. Or are you happy covering that through the acceptance specs?

Reply:
Yeah these tests do more than it looks like.
As you can see we reference 'subject.list_health_checks.health_checks', and expect this to be empty.

This is a stubbed out AWS response object that we are asserting on, meaning that it successfully called the API.

Same with 'subject.get_health_check_status(health_check_id: 'abc').health_check_observations', if the delegation didn't happen correctly, it would blow up.